easybuild/easyconfigs/a/ABySS/ABySS-1.3.7-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'ABySS'
+version = '1.3.7'
+versionsuffix = '-Python-2.7.9'
+
+homepage = 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'
+description = """Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': True}
+
+# eg. http://www.bcgsc.ca/downloads/abyss/abyss-1.3.4.tar.gz
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.bcgsc.ca/downloads/abyss/']
+
+dependencies = [('Boost', '1.58.0', versionsuffix)]
```

Comment:
``sparsehash``  is indeed recommended as it will reduce memory usage. Just added this dependency to the pull request.